src/anomalib/models/image/vlm_ad/backends/ollama.py

```diff
@@ -36,16 +36,30 @@
 
 import logging
 from pathlib import Path
+from typing import TYPE_CHECKING
 
 from lightning_utilities.core.imports import module_available
 
 from anomalib.models.image.vlm_ad.utils import Prompt
 
 from .base import Backend
 
-if module_available("ollama"):
+if TYPE_CHECKING or module_available("ollama"):
     from ollama import Image, chat
 else:
+
+    class Image:
```

Comment:
Maybe we can have a base class for all the import errors? Then we can do something like
```python
else:
    Image = OptionalImport(pip_dependency="ollama", anomalib_dependency="vlm")
```
and

```python
else:
    Image = OptionalImport(pip_dependency="tensorboard", anomalib_dependency="loggers")
```

Just a thought

Reply:
https://github.com/open-edge-platform/anomalib/pull/2954/commits/3d9b6932f64c17182e476476638b13587298de86